sgf/parser.go

```diff
@@ -0,0 +1,174 @@
+package sgf
```

Comment:
Probably package for parser should be parse or parser.

Reply:
I made an attempt at cleaner packaging by making scanner, parser, and game directories inside of "internal"